### DISCUSSION TITLE
Suggestion to improve the ui of the switcher buttons in the Elementor editor

### DISCUSSION THREAD

**Discussion by Michaelgimii** [elementor/elementor#21578] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Hi @nicholaszein

I had two suggestions to improve the ui on the switcher buttons, please check them if possible.

1- Reduce the width of the switcher button to 40px.

2- Delete the text in the switcher buttons (yes, no, show, hide). Being on or off in the switcher buttons is easy for users to understand. Therefore, there is no need for text.

### Describe the solution you'd like

In the image below, I have applied the above two items in Elementor code to show you. I hope you consider it🙏

![switcher](https://user-images.githubusercontent.com/88034902/223399000-955d9ff7-3f47-4276-b4f2-1e6f91035eb2.png)

thanks!

**Comment by hitsgee** [comment]
Isn't text needed for accessibility? Elementor seem to be focusing on this a bit more lately. I do agree with you though looks nicer without.

**Comment by Michaelgimii** [comment]
Hi @hitsgee 
To distinguish whether the switcher buttons are active or inactive, no one pays attention to the text inside. Rather, the first criterion for the switcher buttons to be active is its color. Of course, this is my opinion.
But I ask you, until now, did you pay more attention to the text inside it, or its color? For whether the switcher is active or not.
